### PR TITLE
Get rid of the pangeo-forge image

### PR DESCRIPTION
**Pull request by yuvipanda**
No longer needed as of
https://github.com/pangeo-forge/pangeo-forge-runner/pull/90!

This simplifies pangeo-forge maintenance as well, as we no longer have to match versions of python and apache_beam with whatever is going on here.

### CODE REVIEW

**Review by weiji14 — COMMENTED** [review]
Thanks @yuvipanda! This will simplify maintenance going forward, as we won't need to do stuff like https://github.com/pangeo-data/pangeo-docker-images/pull/470 anymore. Just some suggested changes on the main README.md and should be ok.

**Review by scottyhq — CHANGES_REQUESTED** [review]
Thanks @yuvipanda ! unfortunately a few conflicts now that other things have been merged. Are you able to revisit this?

**Comment by weiji14** [comment]
> I'm waiting to hear from @cisaacstern before moving forward. We all agree this is the path forward - the question now is, do we wait for [pangeo-forge/pangeo-forge-runner#90](https://github.com/pangeo-forge/pangeo-forge-runner/pull/90) to be merged before merging this, or can we do that before?

The `forge` docker images will still be archived at https://hub.docker.com/r/pangeo/forge/tags and https://quay.io/repository/pangeo/forge?tab=tags, and available for download for the forseeable future right? Should be ok to merge this PR now (once the conflicts have been resolved), since it sounds like the pangeo-forge-runner will be moving to Apache Beam supported docker images eventually.

**Comment by cisaacstern** [comment]
> The forge docker images will still be archived at https://hub.docker.com/r/pangeo/forge/tags and https://quay.io/repository/pangeo/forge?tab=tags, and available for download for the forseeable future right?

How long is the foreseeable future? As long as the old images will be archived for long enough to ensure we'll have made the transition by the time the archive expires, I'm 👍

File: README.md [hunk 1]
```diff
@@ -18,7 +18,6 @@ Images are hosted on [DockerHub](https://hub.docker.com/u/pangeo) and on [Quay.i
 | [pangeo-notebook](pangeo-notebook/packages.txt) | base-notebook + core earth science analysis packages | ![](https://img.shields.io/docker/image-size/pangeo/pangeo-notebook?sort=date) | ![](https://img.shields.io/docker/pulls/pangeo/pangeo-notebook?sort=date)
 | [pytorch-notebook](pytorch-notebook/packages.txt) | pangeo-notebook + GPU-enabled pytorch | ![](https://img.shields.io/docker/image-size/pangeo/pytorch-notebook?sort=date) | ![](https://img.shields.io/docker/pulls/pangeo/pytorch-notebook?sort=date)
 | [ml-notebook](ml-notebook/packages.txt) | pangeo-notebook + GPU-enabled tensorflow2 | ![](https://img.shields.io/docker/image-size/pangeo/ml-notebook?sort=date) | ![](https://img.shields.io/docker/pulls/pangeo/ml-notebook?sort=date)
-| [forge](ml-notebook/packages.txt) | pangeo-notebook + [Apache Beam](https://beam.apache.org/) support| ![](https://img.shields.io/docker/image-size/pangeo/forge?sort=date) | ![](https://img.shields.io/docker/pulls/pangeo/forge?sort=date)
 
 *Click on the image name in the table above for a current list of installed packages and versions*
```
[the remark this answers is not in the document]
Comment (yuvipanda) [reply README.md:23]: @weiji14 done!

**Comment by yuvipanda** [comment]
Thanks for chiming in, @cisaacstern!

I've resolved the merge conflicts, and updated the README, @weiji14!

**Comment by weiji14** [comment]
> > The forge docker images will still be archived at https://hub.docker.com/r/pangeo/forge/tags and https://quay.io/repository/pangeo/forge?tab=tags, and available for download for the forseeable future right?
> 
> How long is the foreseeable future? As long as the old images will be archived for long enough to ensure we'll have made the transition by the time the archive expires, I'm 👍

Well, there isn't an [expiry date](https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/use_red_hat_quay/working_with_tags#tag-expiration) set for the quay.io hosted images (see Expires = Never in screenshot below), so it should be until they change their terms and conditions :slightly_smiling_face:

image

**Review by weiji14 — APPROVED** [review]
Thanks @yuvipanda! Will simplify maintenance going forward.

**Comment by cisaacstern** [comment]
Awesome LGTM! Thanks all!

**Review by scottyhq — APPROVED** [review]

**Pull request merged**